src/app/shared/interfaces/station.ts

```diff
@@ -13,4 +13,6 @@ export interface Station {
     characteristics: Array<GageCharacteristic>;
     statistics: Array<GageStatistic>;
     citations: Array<Citation>;
+    filter?;
+    sort?;
```

Comment:
Again, just curious what this is here for

Reply:
No idea what these are doing here. I must have started something and forgotten to remove these.